### ISSUE TITLE
Allow Omni to search URLs as well as Tab Titles.

### ISSUE THREAD

**Issue by aekanshd** [alyssaxuu/omni#96]
As far as I have seen, right now only Tab Titles are searched (along with actions, bookmarks, etc). I want to include the functionality from Chrome's native "Search Tabs..." feature that allows to search for URLs as well as Titles. This will help a lot as many websites have a single title for every page making it hard to differentiate.

P.S.: Chrome's native search tab feature also allows to search for recently closed tabs, so that can be looked into as well but the above is a priority.

**Comment by buttercannfly** [comment]
Yeah it;s implemented now, in https://github.com/buttercannfly/AIpex
![image](https://github.com/user-attachments/assets/483bba48-c8d5-4520-9550-874570be85ae)
